src/macaron/malware_analyzer/pypi_heuristics/metadata/wheel_absence.py

```diff
@@ -64,21 +67,45 @@ def analyze(self, pypi_package_json: PyPIPackageJsonAsset) -> tuple[HeuristicRes
                 logger.debug(error_msg)
                 raise HeuristicAnalyzerValueError(error_msg)
 
-        release_files: list[JsonType] = []
+        file_server_links: list[JsonType] = []
+        inspector_links: list[JsonType] = []
         wheel_present: bool = False
 
         try:
             for release_metadata in releases[version]:
                 if release_metadata["packagetype"] == self.WHEEL:
                     wheel_present = True
 
-                release_files.append(release_metadata["filename"])
+                name = json_extract(pypi_package_json.package_json, ["info", "name"], str)
+                if name is None:
+                    error_msg = "There is no 'name' field for this package."
+                    logger.debug(error_msg)
+                    raise HeuristicAnalyzerValueError(error_msg)
+
+                # include the pypi inspector link, which uses the same suffix of
+                # packages/{blake2b_256}/file_name
+                inspector_prefix = f"{self.INSPECTOR_PREFIX}{name.lower()}/{version}/"
+                inspector_link = release_metadata["url"].replace(self.PYPI_PREFIX, inspector_prefix)
```

Comment:
Although it may never happen but would it be better to ensure that we always replace `self.PYPI_PREFIX` as a prefix, but not as a substring (i.e in the middle of `release_metadata["url"]`)?

Reply:
I've refactored it instead so that it directly conforms to pypi inspector's source code instead of replacing strings now. See my comment below for this.